src/main/java/de/cerus/jdasc/command/ApplicationCommand.java

```diff
@@ -41,6 +41,19 @@ private void validate() {
         }
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof ApplicationCommand)) return false;
+        ApplicationCommand that = (ApplicationCommand) o;
+        return com.google.common.base.Objects.equal(getName(), that.getName()) && com.google.common.base.Objects.equal(getDescription(), that.getDescription()) && com.google.common.base.Objects.equal(getOptions(), that.getOptions());
+    }
+
+    @Override
+    public int hashCode() {
+        return com.google.common.base.Objects.hashCode(getName(), getDescription(), getOptions());
+    }
+
```

Comment:
Was it necessary to override these two methods?

Reply:
yes otherwise they won't compare correctly